packages/circuit-ui/components/TopNavigation/components/ProfileMenu/ProfileMenu.tsx

```diff
@@ -0,0 +1,184 @@
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { useState, HTMLProps } from 'react';
+import { css } from '@emotion/core';
+import { ChevronDown } from '@sumup/icons';
+
+import styled, { StyleProps } from '../../../../styles/styled';
+import { hideVisually, navigationItem } from '../../../../styles/style-mixins';
+import Avatar, { AvatarProps } from '../../../Avatar';
+import Body from '../../../Body';
+import Popover, { PopoverProps } from '../../../Popover';
+
+const profileWrapperStyles = ({ theme }: StyleProps) => css`
+  padding: ${theme.spacings.kilo};
+  border-left: ${theme.borderWidth.kilo} solid ${theme.colors.n200};
+
+  ${theme.mq.mega} {
+    padding: ${theme.spacings.bit} ${theme.spacings.mega};
+  }
+`;
+
+const ProfileWrapper = styled.button(navigationItem, profileWrapperStyles);
+
+const profileAvatarStyles = ({ theme }: StyleProps) => css`
+  width: ${theme.iconSizes.mega};
+  height: ${theme.iconSizes.mega};
+
+  ${theme.mq.mega} {
+    width: ${theme.iconSizes.giga};
+    height: ${theme.iconSizes.giga};
+  }
+`;
+
+const ProfileAvatar = styled(Avatar)(profileAvatarStyles);
+
+const profileNameStyles = ({ theme }: StyleProps) => css`
+  ${theme.mq.untilMega} {
+    ${hideVisually()};
+  }
+
+  ${theme.mq.mega} {
+    margin: 0 ${theme.spacings.kilo};
+    max-width: 20ch;
+    text-overflow: ellipsis;
+  }
+`;
+
+const ProfileName = styled.div(profileNameStyles);
+
+const userNameStyles = css`
+  display: block;
+  max-width: 132px;
+  overflow: hidden;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+`;
+
+const UserName = styled(Body)(userNameStyles);
+
+const chevronStyles = ({ theme }: StyleProps) => css`
+  display: none;
+
+  ${theme.mq.mega} {
+    display: block;
+
+    button[aria-expanded='true'] & {
+      transform: rotate(180deg);
+    }
+  }
+`;
+
+const Chevron = styled(ChevronDown)(chevronStyles);
+
+interface ProfileProps extends HTMLProps<HTMLButtonElement> {
+  /**
+   * A description of the button which opens the profile menu.
+   */
+  profileLabel: string;
+  /**
+   * A user's profile photo.
+   */
+  userAvatar?: AvatarProps;
+  /**
+   * A user's name. Strings longer than 20 characters are truncated.
+   */
+  userName: string;
+  /**
+   * An optional user id such as the SumUp merchant code.
+   */
+  userId?: string;
+  /**
+   * Whether the associated popover is open.
+   */
+  isOpen?: boolean;
+  /**
+   * Whether the profile page is the currently active page.
+   */
+  profileIsActive?: boolean;
```

Comment:
Design question: when this is true, should the active background color transform to a darker shade of blue rather than the gray? Because right now it goes blue (default) 👉 gray (active) 👉 blue (popover opeen)

Reply:
We don't really have a blue anymore that would be suited for this use case. `b40` is closest but quite a bit darker than the current `n30`. 

<img width="465" alt="Screenshot 2021-08-03 at 18 59 33" src="https://user-images.githubusercontent.com/11017722/128056064-c6d09cd0-1a6d-4940-a9df-67a83eb77269.png">

When I click it, I don't even see the `active` state or it's so brief, it's not noticeable. I say 🤷🏻‍♂️